procdiscovery/pkg/inspectors/dotnet/dotnet.go

```diff
@@ -16,14 +16,16 @@ const (
 	dotnet = "DOTNET"
 )
 
-func (d *DotnetInspector) Inspect(p *process.Details) (common.ProgrammingLanguage, bool) {
+func (d *DotnetInspector) Inspect(p *process.Details) (common.ProgramLanguageDetails, bool) {
+	var programLanguageDetails common.ProgramLanguageDetails
 	data, err := os.ReadFile(fmt.Sprintf("/proc/%d/environ", p.ProcessID))
```

Comment:
this part can be done with the new API added in this PR - here it is used to determine the runtime itself (dotnet)